src/fabian_nemazi_ex/ex05/walker_sim.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Fabian Nemazi'
+__email__ = 'fabinema@nmbu.no'
+
+
+import random as rd
+
+
+class Walker:
+    def __init__(self, initial_pos, home_pos):
+        self.start = initial_pos
+        self.end = home_pos
+        self.steps = 0
+
+    def move(self):
+        move = (2 * rd.randint(0, 1)) - 1
+        self.start = self.start + move
+        self.steps += 1
+        # print(self.steps)
+        # print(self.get_position())
+
+    def is_at_home(self):
+        return self.start == self.end
+
+    def get_position(self):
+        return self.start
+
+    def get_steps(self):
+        return self.steps
+
+
+class Simulation:
+    def __init__(self, start, home, seeds):
+        self.start = start
+        self.home = home
+        self.seed = rd.seed(seeds)
+
+    def single_walk(self):
+        rd.seed(self.seed)
```

Comment:
Remove this line. You should set the random seed only once at the beginning of the whole process.